src/dicognito/__main__.py

```diff
@@ -130,10 +130,12 @@ def print_table(version_rows: Sequence[Tuple[str, str]]) -> None:
         help="Set the log level. May be one of DEBUG, INFO, WARNING, ERROR, or CRITICAL.",
     )
     parser.add_argument(
-        "--seed",  # currently only intended to make testing easier
-        help="The seed to use when generating random attribute values. Primarily "
-        "intended to make testing easier. Best anonymization practice is to omit "
-        "this value and let dicognito generate its own random seed.",
+        "--seed",
+        help="The seed to use when generating anonymized attribute values. "
+        "If the same value is supplied for subsequent dicognito invocations, then "
+        "the same input objects will result in consistent anonymized results."
+        "Omitting this value allows dicognito to generate its own random seed, which "
+        "may be slightly more secure, but does not support reproducible anonymization.",

```

Comment:
@annawoodard, based on comments in #126, I thought you might have an opinion about the reworded help for the `--seed` parameter. (Or anything else that strikes your eye.)

If you've a complaint or suggestion, I'm keen to hear it. That being said, no pressure. If you don't want to or don't have time to comment, I'll manage.

I'll leave the review open for a week, unless I hear something (even a request for extension).